### ISSUE TITLE
SPM Support

### ISSUE THREAD

**Issue by naivalf27** [RxSwiftCommunity/RxCoreData#51]
Swift package manager support would be a nice to have. I create PR for this.

**Comment by DaveDude** [comment]
I agree - having SPM support would be good.  @naivalf27 thx for creating the PR - I will be using that commit in the meantime until this PR gets accepted.
